index.d.ts

```diff
@@ -0,0 +1,14 @@
+import { EmojiIdentifierResolvable, Message, MessageEmbed, MessageReaction, ReactionCollectorOptions, User } from "discord.js";
+
+interface PaginationOptions extends ReactionCollectorOptions {
+  deleteOnEnd?: boolean,
+  useUtil?: boolean,
+  emojiList?: EmojiIdentifierResolvable[],
+  collectorFilter?(reaction: MessageReaction, user: User): boolean | Promise<boolean>
+  footerResolver?(pageIndex: number, pagesLength: number): string,
+  pageResolver?(msg: Message, pages: MessageEmbed[], emojiList: string[], currentPageIndex: number, reaction: MessageReaction): number | Promise<number>
+}
+
+declare function paginationEmbed (msg: Message, pages: MessageEmbed[], paginationOptions?: PaginationOptions): Promise<Message>
```

Comment:
Trailing semicolon here for consistency, as well as removing the space between `paginationEmbed` and its parameters.